### PR TITLE
`word-break` styles - DE47252

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
[defect](https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fdefect%2F626939591737)

### CODE REVIEW

File: src/components/rubric-criteria-group.js [hunk 1]
```diff
@@ -91,6 +91,10 @@ class RubricCriteriaGroup extends composeMixins(
 					pointer-events: auto;
 				}
 
+				th {
+					word-break: break-word;
```
Comment (imcdonald-d2l) [inline src/components/rubric-criteria-group.js:95]: | Before | After |
| --- | --- |
| th-before | th-after |

File: src/components/editor/rubric-description-editor.js [hunk 2]
```diff
@@ -55,6 +55,7 @@ class RubricDescriptionEditor extends composeMixins(
 					flex-direction: column;
 					width: 100%;
 					min-height: 4rem;
+					word-break: break-word;
```
Comment (imcdonald-d2l) [inline src/components/editor/rubric-description-editor.js:58]: | Before | After |
| --- | --- |
| overall-description-before | overall-description-after |

**Comment by devpow112** [comment]
Not something we need to do now but might be good to add a long string to the vdiff data so we can make sure this doesn't come back.

**Review by devpow112 — APPROVED** [review]

**Comment by devpow112** [comment]
> Not something we need to do now but might be good to add a long string to the vdiff data so we can make sure this doesn't come back.

Added an issue so I don't forget https://github.com/Brightspace/d2l-rubric/issues/1643.

**Pull request merged**